x/fundraising/types/params.go

```diff
@@ -12,9 +12,11 @@ import (
 // Parameter store keys.
 var (
 	KeyAuctionCreationFee = []byte("AuctionCreationFee")
+	KeyPlaceBidFee        = []byte("PlaceBidFee")
 	KeyExtendedPeriod     = []byte("ExtendedPeriod")
 
 	DefaultAuctionCreationFee = sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(100_000_000)))
+	DefaultPlaceBidFee        = sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(0)))
```

Comment:
This will eventually set the `DefaultPlaceBidFee` to just `sdk.Coins{}`.
```suggestion
	DefaultPlaceBidFee        = sdk.Coins{}
```